src/pages/api/index.js

```diff
@@ -7,15 +9,32 @@ export default function handler(req, res) {
 
   const rooms = Object.keys(room_occupancy[currentDay]).sort();
 
+  const fuseOptions = {
+    threshold: 0.35,
+  };
+
   const fliteredRooms = rooms.filter((room) => {
-    const hasRoomName = room.includes(searchText.toUpperCase());
     const hasAvailableSlot =
       timeSlot == "ALL" ? true : room_occupancy[currentDay][room][timeSlot];
     const isInTargetCluster =
       cluster == "ALL" ? true : cluster_data[room] == cluster;
 
-    return hasRoomName && hasAvailableSlot && isInTargetCluster;
+    return hasAvailableSlot && isInTargetCluster;
   });
 
-  return res.status(200).json(fliteredRooms);
+  const fuse = new Fuse(fliteredRooms, fuseOptions);
```

Comment:
```suggestion
  const fuse = new Fuse(fliteredRooms, {
     threshold: 0.35,
   });
```
Since there is only one option for configuration, would be better to put it inline instead of extracting it out